bundles/framework/printout/view/BasicPrintout.js

```diff
@@ -298,8 +299,15 @@ Oskari.clazz.define('Oskari.mapframework.bundle.printout.view.BasicPrintout',
 
             contentPanel.append(mapTitle);
 
+            me.areLayersWithTimeSeriesSelected =
```

Comment:
Change this to function scoped let/const instead so we don't rely on a side-effect later on.

Reply:
Fixed as proposed.